src/Search/SearchHeader.tsx

```diff
@@ -1,9 +1,48 @@
+import {StackNavigationOptions} from '@react-navigation/stack'
 import React from 'react'
+import {StyleSheet, TouchableOpacity, TouchableOpacityProps} from 'react-native'
 
+import {Icon} from '../components'
 import {SearchBar} from './SearchBar'
 import {useSearch} from './SearchContext'
 
-export interface Props {
+export const useSearchHeaderOptions = ({placeHolderText, title}) => {
```

Comment:
until search is used in multiple places, we may want to keep it colocated

Reply:
@jemubm is splitting it up.

Reply:
needed for send